lib/src/loader.ts

```diff
@@ -88,3 +87,40 @@ export function hCaptchaLoader(params: ILoaderParams = { cleanup: true }): Promi
     return Promise.reject(new Error(SCRIPT_ERROR));
   }
 }
+
+
+export function hCaptchaLoader(params: ILoaderParams = { cleanup: true }) {
```

Comment:
This is a good first step to getting this working! We need to tighten up the readability here before merging.
- We can remove the `then` and `catch` with a simple `try / catch`.
- Make the function names more meaningful then `hCaptchaLoader` and `hCaptchaLoaderPromise`.
  - How about we change `hCaptchaLoaderPromise` to something like: `hCaptchaScript`, `hCaptchaApi`, etc.
- You can reduce passing `resolve`, `reject`, and `retries` to another function by moving some of the functionality around. 
  - For instance, `retries` is already scoped to the function that attemptLoading is being called in. No need to pass that variable into this function.
  - Moving the entire retry login to be within the promise returned will also allow for those callbacks to be available to be used by attemptLoading without needing to pass it in.
- `attemptLoading` is a bit awkward phrasing, doesn't really convey they we can load anything. I would consider calling this `loadScript`.

Reply:
Example code:
```js
const RetryCount = 2;

async function loadScript(params, retries) {
  const message = retries < RetryCount ? 'Retry loading hCaptcha Api' : 'Exceeded maximum retries';
  const category = 'api';

  try {

    const sentry = initSentry(params.sentry);
    const result = await hCaptchaApi(params, sentry);
    return result;

  } catch (error) {

    sentry.addBreadcrumb({
      category,
      message,
      data: { error: error.message }
    });

    if (retries >= RetryCount) {
      sentry.captureException(error);
      throw error;
    } else {
      retries += 1;
      await loadScript(params, retries);
    }

  }
}


function hCaptchaLoader(params) {
  let retries = 0;
  await loadScript(params, retries);
}
```